shared/oae/api/oae.api.user.js

```diff
@@ -259,6 +259,10 @@ define(['exports', 'jquery', 'underscore', 'oae.api.config'], function(exports,
                 'token': token
             },
             'success': function(data) {
+                // Accepting an invitation will set a verified email if there wasn't one already,
+                // so do that here to avoid giving the user a pop-up
+                require('oae.core').data.me.email = require('oae.core').data.me.email || data.email;
```

Comment:
In what scenario do we need to use the existing email from the me object instead of the one returned in the data object?